test/test_datasets.py

```diff
@@ -490,6 +480,35 @@ def inject_fake_data(self, tmpdir, config):
         return num_images_per_category * len(categories)
 
 
+class ImageNetTestCase(datasets_utils.ImageDatasetTestCase):
+    DATASET_CLASS = datasets.ImageNet
+    REQUIRED_PACKAGES = ['scipy']
```

Comment:
Nit: Although this should not make a difference here, one should always avoid mutable class attributes unless this explicitly needed.